terra2/Makefile

```diff
@@ -0,0 +1,65 @@
+bridge_SOURCE=wormhole
+token_bridge_SOURCE=token_bridge_terra
+
+
+-include ../Makefile.help
+
+.PHONY: artifacts
+## Build contracts and copy to terra2
+artifacts:
+	cd ../cosmwasm && make artifacts
+	cp -r ../cosmwasm/artifacts .
+
+
+payer-$(NETWORK).json:
+	$(error Missing private key in payer-$(NETWORK).json)
+
+.PHONY: deploy/bridge
+## Deploy core bridge
+deploy/bridge: bridge-code-id-$(NETWORK).txt
+
+.PHONY: deploy/token_bridge
+## Deploy token bridge
+deploy/token_bridge: token_bridge-code-id-$(NETWORK).txt
+
+%-code-id-$(NETWORK).txt: check-network tools/node_modules payer-$(NETWORK).json artifacts
+	@echo "Deploying artifacts/$($*_SOURCE).wasm on $(NETWORK)"
+	@node tools/deploy_single.js \
+		--network $(NETWORK) \
+		--artifact artifacts/$($*_SOURCE).wasm \
+		--mnemonic "$$(cat payer-$(NETWORK).json)" \
+		| grep -i "code id" | sed s/[^0-9]//g \
+		> $@
+	@echo "Deployed at code id $$(cat $@) (stored in $@)"
+
+tools/node_modules: tools/package-lock.json
+	cd tools && npm ci
+
+LocalTerra:
+	git clone --depth 1 https://www.github.com/terra-money/LocalTerra
+
+test/node_modules: test/package-lock.json
+	cd test && npm ci
+
+.PHONY: unit-test
+## Run unit tests
+unit-test:
+	cargo test -p wormhole-bridge-terra-2
+	cargo test -p token-bridge-terra-2
+	cargo test -p cw20-wrapped-2
```

Comment:
It doesn't look like there is any rust code in this directory so this is unnecessary.

Reply:
fixed, thanks